backend/internal/controller/adapter/dispatcher.go

```diff
@@ -58,6 +62,9 @@ func (dispMsg dispatcherMsg) getMessageCategory() MsgCategory {
 	if dispMsg.Goal != 0 {
 		return MsgGoal
 	}
+	if dispMsg.MsgType == "" {
```

Comment:
Consider changing `""` to something like `msgType.empty`

Reply:
We get an empty `dispMsg.MsgType` when dispatcher sends to us the ball location therefore we compare it to `""`.